File: Saatvik Srivastava/Day 7/Day7B.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int res =0;
        for(int i=0;i<nums.size();i++){
            if(nums[i] !=0){
                nums[res] = nums[i];
                res++;
            }
        }
        for(int i =res;i<nums.size();i++){
            nums[res] = 0;
            res++;
        }
    }
};

File: Mukul Bhalla/Day 7/Day 7 B.py
class Solution(object):
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
     
        for i in nums:
            if(i==0):
                nums.remove(0)
                nums.append(0)
